modules/bullet/generic_6dof_joint_bullet.cpp

```diff
@@ -43,6 +43,12 @@
 Generic6DOFJointBullet::Generic6DOFJointBullet(RigidBodyBullet *rbA, RigidBodyBullet *rbB, const Transform &frameInA, const Transform &frameInB) :
 		JointBullet() {
 
+	for (int i = 0; i < 3; i++) {
```

Comment:
This has been fixed with a different code which is a bit cleaner on master:
```
for (int i = 0; i < 3; i++) {
	for (int j = 0; j < PhysicsServer3D::G6DOF_JOINT_FLAG_MAX; j++) {
		flags[i][j] = false;
	}
}
```

Also, I'm wondering if `memset` would be better in this case. I don't know if all compilers would optimize this loop properly.